data/core/units/elves/Avenger.cfg

```diff
@@ -17,9 +17,9 @@
     {AMLA_DEFAULT}
     cost=53
     usage=mixed fighter
-    description= _ "The curious name of the elvish ‘Avengers’ comes from a tactic often employed by these master rangers. The enemy is allowed to break through a feint defense, and when the vulnerable troops behind the front line follow, these archers break cover and attack, cutting the supply lines and surrounding the enemy in one fell stroke.
+    description= _ "The curious name of the Elvish ‘Avengers’ comes from a tactic often employed by these master rangers. The enemy is allowed to break through a feint defense and when the vulnerable troops behind the front line follow, these archers break cover and attack, cutting supply lines and surrounding the enemy in one fell stroke. This has, at times, been interpreted as a form of vengeance for their brethren lost earlier in the battle. While not so base in design, it is not at all inaccurate.
 
-This has, at times, been interpreted as a form of vengeance for their brethren lost earlier in the battle. Although not so base in design, it is not at all inaccurate."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}
+Even more than other elves, Avengers are a part of the realm of the forests, spending most their time living among the arboreal creatures and learning the hidden secrets in the woods. Spending most of their time outdoors as the keepers of nature, these elves have to propensity to explore the wilds even beyond the boundaries of the forests and are adept at traversing rough terrains like mountains and deserts. Even in such unfamiliar territories, Avengers are already powerful combatants; in their homes in the trees, they are nigh unassailable, capable of assassinating enemies from the shadows without ever having to break stealth. It is this ability which instills the most fear into any invading army, for fighting a force that can barely be tracked or seen in the forests leaves very little room for counterattack and customarily results in crushing defeat."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}
```

Comment:
1. You're missing the "of" in the first instance of "spending most of their time".
2. You've used "spending most of their time" twice in succession, which sounds a bit... awkward I guess?